libs/crowdfunding/editor/src/lib/edit-crowdfunding-form.tsx

```diff
@@ -0,0 +1,131 @@
+import {
+  CreateCrowdfundingGoalInput,
+  UpdateCrowdfundingInput,
+  getApiClientV2,
+  useCrowdfundingQuery,
+  useUpdateCrowdfundingMutation
+} from '@wepublish/editor/api-v2'
+import {useMemo, useState} from 'react'
+import {useTranslation} from 'react-i18next'
+import {useNavigate, useParams} from 'react-router-dom'
+import {CrowdfundingForm} from './crowdfunding-form'
+import {SingleViewTitle} from '@wepublish/ui/editor'
+import {Form, Message, Schema, toaster} from 'rsuite'
+import {ApolloError} from '@apollo/client'
+
+const showError = (error: ApolloError): void => {
+  toaster.push(
+    <Message type="error" showIcon closable duration={3000}>
+      {error.message}
+    </Message>
+  )
+}
+
+export const EditCrowdfundingForm = () => {
+  const {id} = useParams()
+  const crowdfundingId = id!
+  const navigate = useNavigate()
+  const {t} = useTranslation()
+
+  const closePath = '/crowdfundings'
+
+  const [crowdfunding, setCrowdfunding] = useState<UpdateCrowdfundingInput>({
+    id: crowdfundingId,
+    name: ''
+  })
+
+  const client = useMemo(() => getApiClientV2(), [])
```

Comment:
```suggestion
  const client = getApiClientV2()
```

the function getApiClientV2 is already memoized.